spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/InitializationMethod.scala

```diff
@@ -278,12 +278,23 @@ case class ConstInitMethod(value: Double) extends InitializationMethod {
  *  (http://jmlr.org/proceedings/papers/v9/glorot10a/glorot10a.pdf)
  */
 case object Xavier extends InitializationMethod {
+  private var version2: Boolean = false
+
+  def setVersion2(v: Boolean): this.type = {
+    version2 = v
```

Comment:
when should one use version2?

Reply:
If the model is defined by dnn backend, we should use version2, such as the vgg16 in the example.

Reply:
version2 seems quite confusing, can we have a meaningful name?